File: Data Structures & Algorithms/pow-x-n/submission-0.cpp
class Solution {
public:
    double myPow(double x, int n) {
        if(n<0) return 1.0/myPow(x,-n);

        if(n==1) return x;
        if(n==0) return 1;

        double r = myPow(x, n/2);
        return r*r*(n%2==1? x : 1);
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}

        for i in nums:
            if(freq.get(i,-1) == -1):
                freq[i] = 1;
            else:
                freq[i]+=1;
        

        otpt = [(freq[i],i) for i in freq]
        otpt.sort(reverse=True,key=lambda x: x[0])
        return [i[1] for i in otpt[:k]]
